java/core/src/test/java/com/microsoft/bond/testmodel/A.java

```diff
@@ -285,6 +284,11 @@ protected final void deserializeStructFields(TaggedDeserializationContext contex
         }
 
         @Override
+        protected final void deserializeStructFields(UntaggedDeserializationContext context, A<X, Y> value) throws IOException {
+        }
```

Comment:
Also to B/C/D/E ?

---
In reply to: [130750811](https://github.com/Microsoft/bond/pull/542#discussion_r130750811) [](ancestors = 130750811)